src/DataMigration/DataMigration/help/Get-AzDataMigrationService.md

```diff
@@ -67,7 +67,7 @@ Accept wildcard characters: False
 ```
 
 ### -Name
-Name of Database Migration Service.
+Name of Azure Database Migration Service (classic).
```

Comment:
Parameter description should be changed in source code as well, see https://github.com/Azure/azure-powershell/blob/08d52dc9174d7a5cbcabcac69539b3d35a9c60a1/src/DataMigration/DataMigration/Cmdlets/GetAzureDataMigrationService.cs#L57-L63